promotion/claim.go

```diff
@@ -33,6 +37,8 @@ type Claim struct {
 	DrainedAt        pq.NullTime     `db:"drained_at"`
 	UpdatedAt        pq.NullTime     `db:"updated_at"`
 	ClaimType        *string         `db:"claim_type"`
+	AddressID        *string         `db:"address_id"`
+	TransactionKey   *string         `db:"transaction_key"`
```

Comment:
What do you think about using the TransactionKey as the ID? So we have a common identifier for the claim across systems.

Reply:
The `unique_claim_transaction_key` index is used just to de-dupe. I dunno more to speak about more claim identifiers. The address_id is somewhat similar.